workers/unity/Packages/io.improbable.gdk.buildsystem/Configuration/BuildContext.cs

```diff
@@ -60,7 +61,8 @@ public struct BuildContext
                     WorkerType = workerType, BuildEnvironment = buildEnvironment,
                     ScriptingImplementation = scriptImplementation ??
                         PlayerSettings.GetScriptingBackend(BuildPipeline.GetBuildTargetGroup(targetConfig.Target)),
-                    BuildTargetConfig = targetConfig
+                    BuildTargetConfig = targetConfig,
+                    IosSdkVersion = (targetConfig.Target == BuildTarget.iOS) ? iosSdkVersion : null
```

Comment:
what happens if `iOSSdkVersion` is null?

Reply:
Its a nullable, so we check for its value in the build pipeline. If set, we change target